platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/model/WiringModel.java

```diff
@@ -39,23 +39,15 @@ public interface WiringModel extends Startable, Stoppable {
      *
      * @param platformContext the platform context
      * @param time            provides wall clock time
+     * @param defaultPool     the default fork join pool, schedulers not explicitly assigned a pool will use this one
      * @return a new wiring model instance
      */
     @NonNull
-    static WiringModel create(@NonNull final PlatformContext platformContext, @NonNull final Time time) {
-        return new StandardWiringModel(platformContext.getMetrics(), time);
-    }
-
-    /**
-     * Build a new wiring model instance.
-     *
-     * @param metrics provides metrics
-     * @param time    provides wall clock time
-     * @return a new wiring model instance
-     */
-    @NonNull
-    static WiringModel create(@NonNull final Metrics metrics, @NonNull final Time time) {
-        return new StandardWiringModel(metrics, time);
+    static WiringModel create(
```

Comment:
Overloading this method with the default pool seems like a better approach

Reply:
I'm not a fan of using overloaded constructors for optional arguments. Very quickly the constructors for a class grow in count and complexity. IMO, the best way of handling optional parameters is to use the builder model. 

If this is the path we decide to go down, I can open up a ticket and change this to a builder pattern as a stand alone PR. It's a small amount of work, probably a 1 or 2 point task, and I can get it done this week.

Reply:
Marking this as resolved since it's minor, once we work out our plan I'm willing to make changes immediately as a follow up.